src/libraries/Microsoft.Extensions.DependencyModel/src/Microsoft.Extensions.DependencyModel.csproj

```diff
@@ -25,11 +31,17 @@ Microsoft.Extensions.DependencyModel.DependencyContext</PackageDescription>
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Buffers" Version="$(SystemBuffersVersion)" />
-    <PackageReference Include="System.Memory" Version="$(SystemMemoryVersion)" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Text.Encodings.Web\src\System.Text.Encodings.Web.csproj" />
     <ProjectReference Include="$(LibrariesProjectRoot)System.Text.Json\src\System.Text.Json.csproj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
```

Comment:
> @ViktorHofer - do you know - should we only use this in the src project, and not in the ref project? Looking at the existing Extensions.* libraries, we only use it in the src projects, but use ProjectReference in the ref projects.

Yes for now please only use that setting in source projects. For reference assemblies we want P2Ps to continue to exist - at least until reference projects are deleted for OOBs (which will happen very soon).

More important, if a configuration is added to a source project, please make sure that it is also added to the reference project. That makes makes sure that there is no assembly universe mismatch and it will help with the ref<-->src convergance. It also avoids going through the netstandard shim.